app/assets/javascripts/discourse/app/lib/load-script.js

```diff
@@ -89,9 +86,12 @@ export default function loadScript(url, opts) {
     });
 
     const cb = function (data) {
-      if (opts && opts.css) {
-        $("head").append("<style>" + data + "</style>");
+      if (opts?.css) {
+        const style = document.createElement("style");
+        style.innerHTML = data;
```

Comment:
Does it need to be innerHTML instead of innerText here ? Mostly curious...

Reply:
`innerText` should work and is probably better - browser won't try to parse that CSS as HTML. Changing!